build/common.props

```diff
@@ -4,7 +4,8 @@
     <TargetFramework>netstandard2.1</TargetFramework>
     <LangVersion>latest</LangVersion>
     <MajorVersion>4</MajorVersion>
-    <MinorVersion>30</MinorVersion>
+	<MinorVersionPrefix Condition="'$(MinorVersionPrefix)' == ''">10</MinorVersionPrefix>
+    <MinorVersion>$(MinorVersionPrefix)30</MinorVersion>
```

Comment:
nit: indentation

Reply:
wait I don't know why it shows up as indented here when it's not indented in the actual code

<img width="700" alt="image" src="https://github.com/Azure/azure-functions-host/assets/37918412/fc9ec906-3930-44a3-bff0-47bd0a1a11b8">

Reply:
tabs vs spaces?

Reply:
fixed!